doc/policy/mempool-replacements.md

```diff
@@ -74,3 +76,6 @@ This set of rules is similar but distinct from BIP125.
 
 * RBF enabled by default in the wallet GUI as of **v0.18.1** ([PR
   #11605](https://github.com/bitcoin/bitcoin/pull/11605)).
+
+* Full replace-by-fee enabled as a configurable mempool policy as of **v24.0** ([PR
```

Comment:
I think it would be better to refer to this as "general full RBF" in comparison to "opt-in full RBF". The "full" in both cases stands for how much of the transaction can be replaced: we allow _full replacement_ of the transaction instead of requiring that the replacement transaction recreating all outputs the original transaction had (which was a competing approach back then).

```suggestion
* General full replace-by-fee enabled as a configurable mempool policy as of **v24.0** ([PR
```

Reply:
I believe the existing "full (opt-in) RBF" was named so because of its contrast to an earlier proposal of "first-seen-safe RBF".

Reply:
Oh I see about the FFS vs full definition. Is there any chance there are two meanings https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2017-January/013471.html ?

Reply:
After more discussion here and the PR for the upcoming Optech newsletter, I retract my suggestion. I changed my opinion because I realized that FSS RBF is no longer on top of people's mind and from today's perspective "opt-in RBF" and "full RBF" are sufficiently established that it's clear what is meant.